app/code/core/Mage/Sitemap/Model/Sitemap.php

```diff
@@ -146,6 +146,13 @@ public function generateXml()
         $date    = Mage::getSingleton('core/date')->gmtDate('Y-m-d');
         $baseUrl = Mage::app()->getStore($storeId)->getBaseUrl(Mage_Core_Model_Store::URL_TYPE_LINK);
 
+        Mage::dispatchEvent('sitemap_urlset_generating_before', array(
```

Comment:
Why did you move the location of this event?

Reply:
I had mentioned moving it in my original review so I suppose that is why. Reflecting back on that comment I don't think the order really matters